eden/model_base.py

```diff
@@ -90,7 +91,7 @@ def vectorizer_init(self, args):
 
     def estimator_init(self, args):
         """Setup the estimator and set of matching parameter choices."""
-        estimator = SGDClassifier(average=True, class_weight='auto', shuffle=True)
+        estimator = SGDClassifier(average=True, class_weight='balanced', shuffle=True)
```

Comment:
Right now the script is throwing a long warning message for each new instance. Somehow 5*20 warnings per each deafult setup run. So it's up to us to decide..